ui/package.json

```diff
@@ -211,7 +211,6 @@
   },
   "resolutions": {
     "cryptiles": "^4.1.2",
-    "d3-shape": "1.3.7",
```

Comment:
🎉

Reply:
Can we also remove `d3-shape` as a dependency assuming that lineal is pulling it in?

Reply:
I wasn't going to remove that until all the other charts are replaced. It looks like vertical-bar-chart and horizontal-bar-chart still import it for now